.github/workflows/std-workflows.yml

```diff
@@ -7,6 +7,9 @@ on:
     paths-ignore:
      - 'helmcharts/**'
      - '.github/**'
+     - 'blackbox-exporter-values.yaml'
```

Comment:
Reason to ignore these paths?

Reply:
No need to generate new docker image, this is values file for helm charts.